webapp/middlewares/queryParser.js

```diff
@@ -0,0 +1,8 @@
+import url from "url"
+import querystring from "querystring"
```

Comment:
The same here - use native implementation. Query is stored in `req.query`, so no need to use 3rd party parsers